### DISCUSSION TITLE
Remove stale bot

### DISCUSSION THREAD

**Discussion by Sytten** [async-graphql/async-graphql#1102] [General]
Hi!

I think stale bots are a bad idea in general in OSS as they arbitrarily close issues and they just lead to people endlessly commenting "not stale". I think it should be removed.

Best :)

**Comment by frederikhors** [comment]
I agree. I can help to moderate the repo like I did in golang's gqlgen project if you want.

**Comment by sunli829** [comment]
I agree 😂
